websites/site-template/gatsby-config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  pathPrefix: '/site-template',
+  siteMetadata: {
+    title: 'CHANGE ME',
+    description: 'CHANGE ME',
+    author: 'commercetools',
+  },
+  plugins: [
+    {
+      resolve: '@commercetools-docs/gatsby-theme-docs',
+      options: {
+        websiteName: 'CHANGE ME',
+        gaTrackingId: 'CHANGE OR REMOVE ME',
```

Comment:
I would remove this if we don't use it. Or leave it but as a comment

Reply:
Right, this is JS and you have comments  - totally forgot about that :-)